internal/rbd/replicationcontrollerserver.go

```diff
@@ -133,12 +138,24 @@ func getMirroringMode(ctx context.Context, parameters map[string]string) (librbd
 // validateSchedulingDetails gets the mirroring mode and scheduling details from the
 // input GRPC request parameters and validates the scheduling is only supported
 // for snapshot mirroring mode.
-func validateSchedulingDetails(parameters map[string]string) error {
+func validateSchedulingDetails(ctx context.Context, parameters map[string]string) error {
 	var err error
 
 	val := parameters[imageMirroringKey]
 
 	switch imageMirroringMode(val) {
+	case imageMirrorModeJournal:
+		// journal mirror mode does not require scheduling parameters
+		if _, ok := parameters[schedulingIntervalKey]; ok {
+			log.WarningLog(ctx, "%s parameter cannot be used with %s mirror mode, ignoring it",
+				schedulingIntervalKey, string(imageMirrorModeJournal))
```

Comment:
nit: no need to convert this to a string, that should be done automatically. It is done in other places already, so not blocking this PR, but might be good to clean it up later.